java/Makefile

```
@@ -460,48 +454,51 @@ export DAIKONDIR
-### Compilation targets
+### Test targets
 ###
 
-# all_directly is not marked as phony
-.PHONY: default compile all all_stop_on_error all_via_daikon
-default: all
-compile: all
-
-# "-S" means do not continue after errors.
-all: all_stop_on_error
+all: daikon
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)